content/references/protocol-reference/data-types/nftoken.md

```diff
@@ -48,64 +48,20 @@ The 16-bit flags, transfer fee fields, the 32-bit `NFTokenTaxon`, and the sequen
 Flags are properties or other options associated with the `NFToken` object.
 
 
-<table>
-  <tr>
-   <td><strong>Flag Name</strong>
-   </td>
-   <td><strong>Flag Value</strong>
-   </td>
-   <td><strong>Description</strong>
-   </td>
-  </tr>
-  <tr>
-   <td><code>lsfBurnable</code>
-   </td>
-   <td><code>0x0001</code>
-   </td>
-   <td>If set, indicates that the issuer (or an entity authorized by the issuer) can destroy the object. The object's owner can <em>always</em> do so.
-   </td>
-  </tr>
-  <tr>
-   <td><code>lsfOnlyXRP</code>
-   </td>
-   <td><code>0x0002</code>
-   </td>
-   <td>If set, indicates that the tokens can only be offered or sold for XRP.
-   </td>
-  </tr>
-  <tr>
-   <td><code>lsfTrustLine</code>
-   </td>
-   <td><code>0x0004</code>
-   </td>
-   <td>If set, indicates that the issuer wants a trustline to be automatically created. The <code>lsfTrustLine</code> field is useful when the token can be offered for sale for assets other than XRP and the issuer charges a <code>TransferFee</code>. If this flag is set, a trust line is automatically created as needed to allow the issuer to receive the appropriate transfer fee. If this flag is not set, an attempt to transfer the NFToken for an asset for which the issuer does not have a trustline fails.
-   </td>
-  </tr>
-  <tr>
-   <td><code>lsfTransferable</code>
-   </td>
-   <td><code>0x0008</code>
-   </td>
-   <td>If set, indicates that this <code>NFToken</code> can be transferred. This flag has no effect if the token is being transferred <em>from</em> the issuer or <em>to</em> the issuer.
-   </td>
-  </tr>
-  <tr>
-   <td><code>lsfReservedFlag</code>
-   </td>
-   <td><code>0x8000</code>
-   </td>
-   <td>This proposal reserves this flag for future use. Attempts to set this flag fail.
-   </td>
-  </tr>
-</table>
-
-
-`NFToken` flags are immutable: they can only be set during the `NFTokenMint` transaction and cannot be changed later.
+| Flag Name       | Flag Value | Description                                   |
```

Comment:
We disagree on the readability of information and ease of editing of HTML vs Markdown. I find the Markdown version harder to read when the lines go past the right margin.

Reply:
I adjusted the spacing of the columns in the Markdown so at least they line up a little better. I also rewrote several of the descriptions to be shorter and follow our new guideline on using "enabled" rather than "set" for flags.